cmd/ecs-deploy-task-container/main.go

```diff
@@ -26,7 +26,7 @@ var services = []string{"app"}
 var environments = []string{"prod", "staging", "experimental"}
 
 // Commands should be the name of the binary found in the /bin directory in the container
-var commands = []string{"save-fuel-price-data"}
+var commands = []string{"ecs-tasks"}
```

Comment:
I bet this has a problem and we'll need to modify to:

```suggestion
var commands = []string{"ecs-tasks save-fuel-price-data", "ecs-tasks send-post-move-survey-email"}
```

Reply:
ok. yeah wasn't sure what to do here.